### DISCUSSION TITLE
Publications section

### DISCUSSION THREAD

**Discussion by davidedellagiustina** [mintyfrankie/awesomeCV-Typst#3] [Ideas]
Hi! Are there plans already for a dedicated "publications" section? It would be ideal if it could parse directly from a BibTeX/Hayagriva file!

**Comment by yunanwg** [comment]
Hi @davidedellagiustina thanks for bringing this up! I haven't had this on mind but it sounds quite an interesting and elegant solution; stay tuned and I will try to make it happen in the near future.

**Reply by davidedellagiustina** [reply]
Thanks!

**Comment by yunanwg** [comment]
Hi @davidedellagiustina, just wanted to let you know that I've pushed [a new commit](https://github.com/mintyfrankie/awesomeCV-Typst/commit/5f5b73aca8dc122bfb0d6e1908f0e93b41290af8) that provide rudimentary support on a Publication section. Feel free to test it out.

However, full customization is not yet supported by current version of Typst (see this issue [#942](https://github.com/typst/typst/issues/942)). I'll wait for later updates to see if we can make this better match the general aesthetics of the CV. 

Any advice or PR welcome for this!

**Reply by davidedellagiustina** [reply]
Super cool! I'll test it out as soon as I can :D
Thanks for the quick update!
